pkg/broker/ingress/errors.go

```diff
@@ -0,0 +1,11 @@
+package ingress
+
+import "errors"
+
+// ErrNotFound is the error when a broker doesn't exist in the configmap.
+// This can happen if the clients specifies invalid broker in the path, or the configmap volume hasn't been updated.
+var ErrNotFound = errors.New("not found")
+
+// ErrIncomplete is the error when a broker entry exists in the configmap but its decouple queue is nil or empty.
+// This should never happen unless there is a bug in the controller.
```

Comment:
This is just a comment, not a suggestion. Looking ahead to future broker and trigger lifecycle, it's possible that a broker entry could have a nil decouple queue if it's been deleted but its triggers still exist and have retry queues. In this case we should probably have a DELETED status for the broker in the targets config.

@yolocs for thoughts. I plan to investigate how PubSub handles deletion of topics and pull subscriptions and write that down (though if someone else wants to do it that's fine too).

Reply:
In that case the code here will have to check the broker status first. For now let's assume the decouple queue is never nil and see what you finds out how pubsub handles deletion. We can change the ingress logic accordingly. SG?

Reply:
@grantr sgtm. It's easy to add additional states.